doc/source/contributing_code.txt

```diff
@@ -131,11 +131,14 @@ all issues have been resolved.
 <li>Programmers begin work on a feature or fix by branching from
 develop.
 
-<li>When branch is ready, it is submitted to code review.
+<li>When a branch is ready, it is submitted to code review.
 
 <li>When code review is complete, and the changes are approved, the PR
 is merged into the develop branch.
 
+<li>Mutliple merges into the develop branch may take place between
+test cycles. (???)
+
```

Comment:
Ideally it would be one at a time because that's the best way to isolate the cause of test fails.  But groups of small PRs can go in at the same time.  Major PRs that touch lots of files and change behavior should go in by themselves.  Its a judgement call.